pokedex-with-bit/app.py

```diff
@@ -20,39 +23,50 @@ def config():
     os.environ['JINA_PORT'] = os.environ.get('JINA_PORT', str(45678))
 
 
-# for index
-def index():
+def index(num_docs: int):
     f = Flow.load_config('flows/index.yml')
+    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC), recursive=True)))
 
     with f:
-        f.index_files(image_src, request_size=64, read_mode='rb', size=num_docs)
+        with TimeContext(f'QPS: indexing {num_docs}', logger=f.logger):
+            f.index_files(IMAGE_SRC, request_size=64, read_mode='rb', size=num_docs)
 
 
-# for search
-def search():
+def query():
     f = Flow.load_config('flows/query.yml')
+    f.use_rest_gateway()
 
+    # no perf measure, as it opens a REST api and blocks
```

Comment:
Check Tobias' PR https://github.com/jina-ai/examples/pull/459 for how to do queries in this case. You can pass them via stdin in the bash shell

Reply:
Thanks for the tip. I have it implemented it for the cases where we read from stdin. But for the rest endpoint queries it is more difficult to automate, right? I could restructure the query to take a path from stdin and then load this example and run the query flow. But it might be more confusing for the reader of the code than it is helpful for the perf measure. What do you think?

Reply:
Ah yes, then maybe it's not worth it